### PR TITLE
chore: update gatling-sbt plugin to v4.1.0

### PR DESCRIPTION
**Pull request by tpetillot**
**Modifications:**
* Update gatling-sbt to 4.1.0
* Document new configurations in build.sbt

**Ref:** MISC-323

### CODE REVIEW

**Review by guilgaly — COMMENTED** [review]
I don't think we should give the detailed explanations here, duplicating the documentation (this was already discussed before with @slandelle last time we updated the samples).

There should only be :
- the first two line with the link to the documentation
- the raw examples, e.g. `// Gatling / enterprisePackageId := "00000000-0000-0000-0000-000000000000"`
- and maybe also keep the recommendation to use the env var for the api token (to make sure we're not encouraging people to commit the token to their source repo)

**Review by slandelle — CHANGES_REQUESTED** [review]
100% agree with @guilgaly . We don't want doc to be duplicated all over the place. Our website is the place to go.

File: build.sbt [hunk 1]
```diff
@@ -9,3 +9,17 @@ scalacOptions := Seq(
 val gatlingVersion = "3.7.3"
 libraryDependencies += "io.gatling.highcharts" % "gatling-charts-highcharts" % gatlingVersion % "test,it"
 libraryDependencies += "io.gatling"            % "gatling-test-framework"    % gatlingVersion % "test,it"
+
+// Enterprise Cloud configurations (https://cloud.gatling.io/)
+// Reference: https://gatling.io/docs/gatling/reference/current/extensions/sbt_plugin/#working-with-gatling-enterprise-cloud
+
+// Alternative (preferred for security reasons):
```
Comment (slandelle) [inline build.sbt:16]: I don't get it. What's this? Why doesn't it go in the aforementioned page?

**Review by slandelle — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (tpetillot) [reply build.sbt:16]: As @guilgaly state:
> and maybe also keep the recommendation to use the env var for the api token (to make sure we're not encouraging people to commit the token to their source repo)

Comment (slandelle) [reply build.sbt:16]: 0 doc here. Everything on the website.

Comment (tpetillot) [reply build.sbt:16]: If so, maybe at least:
```suggestion
// For security purposes, check other API token configuration methods: 
// https://gatling.io/docs/gatling/reference/current/extensions/sbt_plugin/#api-tokens
```

**Pull request merged**